src/extension.ts

```diff
@@ -1,9 +1,40 @@
 import { ExtensionContext } from "vscode";
 import { GodotTools } from "./godot-tools";
-
+import * as path from "path";
+import * as fs from "fs"
+import * as vscode from 'vscode'
 
 let tools: GodotTools = null;
 
+export class DocContent {
+	static docItems: vscode.QuickPickItem[] = []
```

Comment:
We use snake_case in this project to make contributors form godot project comfotable.